docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-installing-in-other-containers-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='_installing_different_containers']
+= Installing {KIE_SERVER} in Other Containers
+
+[id='_decision_server_tomcat']
+== Red Hat JBoss Web Server 2.__X__/3.__X__, Tomcat 8.__X__/9.__X__
```

Comment:
No support of JWS 2 in RHDM 7.0, only version 3

Reply:
Not included in this assembly.